other-languages/en_US/glm-en_US.py

```diff
@@ -0,0 +1,412 @@
+                else:
+                    print(f"{Fore.RED}✕{Fore.RESET} Failed to update the label {Fore.BLUE}{label['name']}{Fore.RESET}: {Fore.YELLOW}{response.status_code}{Fore.RESET}\n{Fore.RED}{response.text}{Fore.RESET}")
+                    return "update error"
+        else:
+            print(f"{Fore.RED}✕{Fore.RESET} Failed to create label {Fore.BLUE}{label['name']}{Fore.RESET}: {Fore.YELLOW}{response.status_code}{Fore.RESET}\n{Fore.RED}{response.text}{Fore.RESET}")
+            return "error"
+    return "successful"
+
+# ---------------------------------------------------------------------------
+
+def copy_labels(source_owner, source_repo, set_owner, set_repo, token, json_file=os.path.join(script_path, "labels-temp.json"), save=False):
+    # 调用get与set函数复制仓库标签
+    # 传入先所有者，再仓库名
+    # 先源仓库，再目标仓库，token，json_file，save
+    if get_labels(source_owner, source_repo, json_file) == "successful":
+        if set_labels(set_owner, set_repo, token, json_file) == "successful":
+            if not save:
+                try:
+                    os.remove(json_file)
+                    return "successful"
+                except Exception as e:
+                    print(f"{Fore.RED}✕{Fore.RESET} Error deleting temporary data file:\n{Fore.RED}{e}{Fore.RESET}")
+                    return "file error"
+    return "function not return successful"
+
+# ---------------------------------------------------------------------------
+
+def main():
+    parser = argparse.ArgumentParser(description='GitHub Labels Manager (GLM), Tools that automatically help you copy repo labels, get repo labels, and empty existing labels.')
+    subparsers = parser.add_subparsers(dest='command', required=True, help='Available command')
+
+    # 命令：get
+    parser_get = subparsers.add_parser('get', help='Get labels')
+    parser_get.add_argument('repo_url', type=str, help='GitHub repo URL')
+    parser_get.add_argument('--save', type=str, help='Location where the label information is saved')
+
+    # 命令：set
+    parser_set = subparsers.add_parser('set', help='Set labels')
+    parser_set.add_argument('repo_url', type=str, help='GitHub repo URL')
+    parser_set.add_argument('--token', type=str, help='GitHub Token')
+    parser_set.add_argument('--json', type=str, help='Label data json file')
+
+    # 命令：copy
+    parser_copy = subparsers.add_parser('copy', help='Copy labels')
+    parser_copy.add_argument('source_repo_url', type=str, help='Source repo URL')
+    parser_copy.add_argument('set_repo_url', type=str, help='Target repo URL')
+    parser_copy.add_argument('--token', type=str, help='GitHub Token')
+    parser_copy.add_argument('--json', type=str, help='Location of the Label data json file (default: labels-temp.json in the glm directory)')
+    parser_copy.add_argument('--save', help='Reserve acquired Label data json file', action='store_true')
```

Comment:
```suggestion
    parser_copy.add_argument('--save', help='Reserve the Label data json file', action='store_true')
```
How about this translation?

Reply:
Just omit the word... Fine.